tests/pages/locators.py

```diff
@@ -96,7 +96,7 @@ class InviteUserPageLocators(object):
         By.CSS_SELECTOR,
         "button[aria-controls=folder_permissions]",
     )
-    SEND_INVITATION_BUTTON = (By.CSS_SELECTOR, "main [type=submit]")
+    SEND_INVITATION_BUTTON = (By.CSS_SELECTOR, "form button")
```

Comment:
When JS is running, there are 2 buttons on this page (assuming it is `/services/<service_id>/users/invite`):

<img width="329" alt="image" src="https://user-images.githubusercontent.com/87140/200561340-1f4f5e52-ab59-47ef-b81e-3f7b2a8d7432.png">

Could do `form button.page-footer__button` instead to differentiate:

```suggestion
    SEND_INVITATION_BUTTON = (By.CSS_SELECTOR, "form button.page-footer__button")
```